src/games/strategy/common/ui/BasicGameMenuBar.java

```diff
@@ -172,10 +171,8 @@ protected void addGameNotesMenu(final JMenu parentMenu) {
     if (notesProperty != null && notesProperty.trim().length() != 0) {
       final String notes = LocalizeHTML.localizeImgLinksInHTML(notesProperty.trim());
       m_gameNotesPane = new SoftJEditorPane(notes);
-      parentMenu.add(SwingAction.of("Game Notes...", e -> {
-        SwingUtilities.invokeLater(new Runnable() {
-          @Override
-          public void run() {
+      parentMenu.add(SwingAction.of("Game Notes...", e ->
+          SwingUtilities.invokeLater(() -> {
```

Comment:
This is a little hard to follow with the lambda chaining. Might be better to split it up some.